.goreleaser.yaml

```diff
@@ -0,0 +1,219 @@
+      - CGO_ENABLED=1
+
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+
+    # Custom ldflags mostly to avoid setting main.date which for some
+    # reason is default
+    ldflags:
+      -w -X main.version={{ .Version }} -X main.signerAppNoTouch= -buildid=
+
+  - id: windows
+    main: ./cmd/tkey-ssh-agent
+    binary: tkey-ssh-agent
+    env:
+      - GOPROXY=https://proxy.golang.org,direct
+      - GOSUMDB=sum.golang.org
+      - CGO_ENABLED=0
+
+    goos:
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+
+    # Custom ldflags mostly to avoid setting main.date which for some
+    # reason is default
+    ldflags:
+      -w -X main.version={{ .Version }} -X main.signerAppNoTouch= -buildid=
+
+    hooks:
+      pre: cd ./cmd/tkey-ssh-agent && go-winres make --arch amd64
+
+  - id: windows-tray
+    main: ./cmd/tkey-ssh-agent-tray
+    binary: tkey-ssh-agent-tray
+    env:
+      - GOPROXY=https://proxy.golang.org,direct
+      - GOSUMDB=sum.golang.org
+      - CGO_ENABLED=0
+
+    goos:
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+
+    # Custom ldflags mostly to avoid setting main.date which for some
+    # reason is default
+    ldflags:
+      -w -H windowsgui -buildid=
+
+    hooks:
+      pre: cd ./cmd/tkey-ssh-agent-tray && go-winres make --arch amd64
+
+universal_binaries:
+  - ids:
+      - darwin
+    replace: true
+    name_template: "tkey-ssh-agent"
+    hooks:
+      post: gon gon.hcl
+
+archives:
+  - format: tar.gz
+    allow_different_binary_count: true
+    # this name template makes the OS and Arch compatible with the results of uname.
+    name_template: >-
+      {{ "tkey-ssh-agent" }}_
+      {{- .Version }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "all" }}universal
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+    # use zip for windows archives
+    format_overrides:
+    - goos: windows
+      format: zip
+    files:
+      - src: system/tkey-ssh-agent.1
+        dst: man
+        strip_parent: true
+
+nfpms:
+  # note that this is an array of nfpm configs
+  - #
+    # ID of the nfpm config, must be unique.
+
+    # Name of the package.
+    package_name: tkey-ssh-agent
+
+    # Your app's vendor.
+    vendor: Tillitis AB
+
+    # Your app's homepage.
+    homepage: https://tillitis.se/
+
+    # Your app's maintainer (probably you).
+    maintainer: Tillitis <hello@tillitis.se>
+
+    # Your app's description.
+    description: |-
+       SSH agent backed by Tillitis TKey
+       tkey-ssh-agent is an alternative SSH agent backed by a private ed25519 key
+       residing in the hardware TKey, a USB stick.
+
+    # Your app's license.
+    license: GPL 2.0
+
+    # Formats to be generated.
+    formats:
+      - apk
+      - deb
+      - rpm
+      - archlinux # Since: v1.13
+
+    dependencies:
+      - pinentry-gnome3 | pinentry
```

Comment:
Alpine does not support "|" to specify dependencies - seems like only Debian and derivatives does. 

It does not have "Pinentry-gnome3" either, but seems to have "pinentry-gnome". 
Probably needs to make an override for Alpine here.

Reply:
Fixed, overrides dependencies for Alpine.